openzaak/src/main/kotlin/com/ritense/openzaak/autoconfigure/OpenZaakAutoConfiguration.kt

```diff
@@ -146,9 +147,19 @@ class OpenZaakAutoConfiguration {
     fun zaakStatusService(
         restTemplate: RestTemplate,
         openZaakConfigService: OpenZaakConfigService,
-        tokenGeneratorService: OpenZaakTokenGeneratorService
+        tokenGeneratorService: OpenZaakTokenGeneratorService,
+        documentService: DocumentService,
+        zaakTypeLinkService: com.ritense.openzaak.service.ZaakTypeLinkService,
+        zaakInstanceLinkService: com.ritense.openzaak.service.ZaakInstanceLinkService
```

Comment:
note: It imports the interface here. not the implementation. 

I find it strange that we dont do this for all other beans

Reply:
We should.